### ISSUE TITLE
[T20][Accessibility][Functional][Unable to resize]

### ISSUE THREAD

**Issue by CernerA** [Mohanme13/Feb-19-Bug-bash#234]
Unable to resize the tab(Annotated display) under past medical and on hovering no tool tip is being displayed.
